doc/rtd/explanation/format.rst

```diff
@@ -141,66 +225,128 @@ Create user data containing 3 shell scripts:
 
     $ cloud-init devel make-mime -a always.sh:x-shellscript-per-boot -a instance.sh:x-shellscript-per-instance -a once.sh:x-shellscript-per-once
 
-``include`` file
-================
 
-This content is an :file:`include` file.
+Cloud config archive
+====================
+
+| **Header:** #cloud-config-archive
+| **Content-Type:** text/cloud-config-archive
+
+Example
+-------
+
+.. code-block:: shell
+
+    #cloud-config-archive
+    - type: "text/cloud-boothook"
+    content: |
+        #!/bin/sh
+        echo "this is from a boothook." > /var/tmp/boothook.txt
+    - type: "text/cloud-config"
+    content: |
+        bootcmd:
+        - echo "this is from a cloud-config." > /var/tmp/bootcmd.txt
+
+Explanation
+-----------
+
+A cloud-config-archive is a way to specify more than one type of data
```

Comment:
Are there any benefits over a MIME multi-part archive? Why would anyone want to use this over MIME?

Reply:
Why would anyone NOT want to use this over MIME? Have you tried building one yourself? Do you like calling helper commands to build your user data? :wink: 

I can add a "for those that prefer to use YAML" to the end of this section. Are you wanting more of an explanation than that in the docs?

Reply:
Yeah, I get it :)

>  It can be seen as an alternative to building a MIME multi-part archive manually.

We present two alternatives, without any recommended / preferred choice or cost / benefit analysis to the end user. If they've tried MIME before, then they'll know, but if not then they have to learn the hard way. Maybe we could save them the trouble?